packages/common/test/compose.test.ts

```diff
@@ -393,6 +393,17 @@ describe("composeContent", () => {
         "should set url"
       );
     });
+    it("public, multi-layer feature service w/o layerId", () => {
+      const layerContent = composeContent(item, { layers });
+      expect(layerContent.layer).toBeUndefined();
+      expect(layerContent.hubId).toBe(item.id);
+      expect(layerContent.type).toBe(item.type);
+      expect(layerContent.family).toBe("map");
+      expect(layerContent.url).toEqual(item.url);
```

Comment:
@sonofflynn89 this addresses https://github.com/Esri/hub.js/pull/745#discussion_r829349011